src/viewer/Viewer.jsx

```diff
@@ -101,6 +106,10 @@ export default class Viewer extends Component {
     const hasNext = currentIndex < fileCount - 1
     // this `expanded` property makes the next/previous controls cover the displayed image
     const expanded = currentFile && currentFile.class === 'image'
+    if (isMobileApp) {
+      StatusBar.styleBlackOpaque()
+      StatusBar.backgroundColorByName('black')
```

Comment:
That's… very black. Can we use the same grey as the viewer ?

Reply:
In fact, I tried to put the logic to cozy-ui in order to have this behavior enabled by default but it seems that the plugin doesn't work very well with css var :/. 

Also : 
> The Android 5+ guidelines specify using a different color for the statusbar than your main app color (unlike the uniform statusbar color of many iOS apps)

So not sure having the same gray is the way to go 

@joel-costa wanted the same a you: https://gyazo.com/81c4f349c8032384a5fb87a95829a1e0

Reply:
Mmh ok, if the guidelines forces us to have a different color, so be it. Still not sure about a pure black though but I'll let @joel-costa decide about that. Otherwise the code LGTM.

Reply:
Ideally, we are supposed to have :
- On iOS : the exact same dark gray
- on Android : a gray darker than the background (I think we had chosen a darker: + 24% with @kosssi )

@Crash-- : in cozy-ui, is it asked for iOS to also have a color darker than the background ?